code/index.html

```diff
@@ -3,28 +3,91 @@
+</head>
 
-    <!-- Signup form -->
+<body>
+   <header class:"hero">
+      <picture>
+        <source srcset="https://images.unsplash.com/photo-1605812830360-f2b3e3a48db4?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" width="20%" media="(max-width: 1024px)" alt="desktop mobile image" />
+        <img class="hero-image" src="https://images.unsplash.com/photo-1605812830455-2fadc55bc4ba?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" alt="hero image" />     
+      </picture>
+     <h1 class="hero-title">Welcome to Dad's Corner</h1>
+     <div class="hero-menu">
+       <nav class="topnav">
+         <a href="#home">Home</a>
+         <a href="#news">News</a>
+         <a href="#contact">Contact</a>
+         <a href="#about">About</a>
+       </nav>
+      </div>
+   </header>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<!-- Intro paragraph -->
+ 
+   <section class="introduction">
+      <h2>Are you a dad on paternity leave? This is the forum for you! Meet other dads, share experiences, have fun play-dates - it all happens here!</h2>
+   </section>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+<!-- The accordion section -->
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+   <section class="information">
+     <div class="accordion" class="container">
+       <h3>FAQ</h3>
+       <div class="question" id="section1"><h2>What is Dad's Corner?</h2></div>
+       <div class="answer" id="answer1"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p></div>
+       <div class="question" id="section2"><h2>How do I create an account?</h2></div>
+       <div class="answer" id="answer2"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p></div>
+       <div class="question" id="section3"><h2>How much does it cost?</h2></div>
+       <div class="answer" id="answer3"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,</p></div>
+       <div class="question" id="section4"><h2>What are the benefits</h2></div>
+       <div class="answer" id="answer4"><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut  labore et dolore magna aliqua. Ut enim ad minim veniam.</p></div>
+     </div>
+
+ <!-- The right image -->
+
+     <div class="dad-son-img" class="container"></div>
```

Comment:
if you don't have anything inside this div you can use a selfclosing tag instead.
Also, if you have to classes on the tag, put them together with a space between
`<div class="dad-son-img container" />`